ibis/backends/tests/test_client.py

```diff
@@ -202,3 +205,128 @@ def test_separate_database(con, alternate_current_database, current_data_db):
     db = con.database(current_data_db)
     assert db.name == current_data_db
     assert tmp_db.name == alternate_current_database
+
+
+def _create_temp_table_with_schema(con, temp_table_name, schema, data=None):
+
+    con.drop_table(temp_table_name, force=True)
+    con.create_table(temp_table_name, schema=schema)
+    temporary = con.table(temp_table_name)
+    assert len(temporary.execute()) == 0
+
+    if data is not None and isinstance(data, pd.DataFrame):
+        con.load_data(temp_table_name, data, if_exists='append')
+        assert len(temporary.execute()) == len(data.index)
+        tm.assert_frame_equal(temporary.execute(), data)
+
+    return temporary
+
+
+@pytest.mark.only_on_backends(
```

Comment:
these are not fully generic tests as the .insert api is not generic. move to the sql alchemy test suite

Reply:
@datapythonista wanted the tests to be present here so that we can later use these tests for all the backends once the implementations are complete.